plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -22,13 +22,17 @@ deploy.preferences.save.error.title=Could not save preferences
 deploy.preferences.save.error.message=Error while saving the preferences: {0}
 error.account.missing.signedin=Select an account.
 error.account.missing.signedout=Sign in to Google.
+error.not.a.file=Path is a directory: {0}
+error.app.yaml.empty: Missing app.yaml path.
```

Comment:
s/:/=/, right?

Reply:
Good catch. Thanks.

Reply:
(It's interesting that tests are passing, so, the colon seems working.)